examples/components/Radar.vue

```diff
@@ -1,5 +1,18 @@
+/**
+  Radar is usually used to describe where we are looking at in 360 panorama viewers or VR projects,
+  inspired by krpano's radar plugin https://krpano.com/tours/bkeller/index.html
+  angle: field of view
+  direction: center angle of the view
+*/
+
 <template>
-  <v-wedge :config='config'></v-wedge>
+<div>
```

Comment:
Do not use `div` in Konva components, this is a DOM elements and here you are in a canvas context. Use `v-group` instead.

Reply:
wow `v-group` is what I needed!